packages/app/obojobo-document-engine/src/scripts/oboeditor/components/navigation/more-info-box.js

```diff
@@ -6,6 +6,7 @@ import React from 'react'
 
 import MoreInfoIcon from '../../assets/more-info-icon'
 import TriggerListModal from '../triggers/trigger-list-modal'
+import ObjectiveListModal from '../objectives/objective-list-modal'
```

Comment:
Looks like the Objective stuff got mixed in with this branch, or this was based off of the Objective branch. Is it possible to remove the Objective stuff from this branch without too much pain?